controllers/controllerComments.js

```diff
@@ -0,0 +1,11 @@
+const {deleteComment} = require ('../models/modelComments')
+
+function deleteCommentById(req, res, next){
+    const {comment_id} = req.params
+    deleteComment(comment_id).then((result) => {
+        res.status(204).send('Comment has been deleted')
```

Comment:
You don't need to send a message for this one. The status 204 and an empty object is fine 🙂